src/handlers/push.ts

```diff
@@ -63,10 +63,11 @@ async function handlePush(context: Context): Promise<void> {
     event_type: repository_dispatch_type,
     client_payload: {
       id: _id.toString(),
+      token: token.data.token,
       ...data,
-      token: token.data.token
+      data: context.payload
```

Comment:
I think naming is a bit confusing because we use the spread operator on `data` on line 67. I wouldn't use a variable and a object property within the same context if they have a different value. Could we either rename the object property or otherwise the data variable?

Reply:
I renamed it to `event`. I thought about `origin_event`or `original_event`, but I think that in the context of a repository dispatch just `event` shouldn't be misleading.